<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoiceHomeMainWindow</class>
 <widget class="QMainWindow" name="VoiceHomeMainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VoiceHome</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:black;</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="HomeButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>580</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/HomeIcon.png</normaloff>:/images/HomeIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="CommandsButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>580</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/ChatIcon.png</normaloff>:/images/ChatIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBoxSvet">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>81</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator:unchecked {
    image: url(:/images/CheckBoxOn.png);
}
QCheckBox::indicator:checked {
    image: url(:/images/CheckBoxOff.png);
}
QCheckBox {
    background-color: rgba(255, 255, 255, 0);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>35</height>
     </size>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="FanLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/IconFan.png</pixmap>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBoxFan">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>90</y>
      <width>81</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator:unchecked {
    image: url(:/images/CheckBoxOn.png);
}
QCheckBox::indicator:checked {
    image: url(:/images/CheckBoxOff.png);
}
QCheckBox {
    background-color: rgba(255, 255, 255, 0);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/CheckBoxOn.png</normaloff>:/images/CheckBoxOn.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>35</height>
     </size>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ChainikLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/IconChainik.png</pixmap>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBoxChainik">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator:unchecked {
    image: url(:/images/CheckBoxOn.png);
}
QCheckBox::indicator:checked {
    image: url(:/images/CheckBoxOff.png);
}
QCheckBox {
    background-color: rgba(255, 255, 255, 0);
}
</string>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="UserCommandButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>360</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(34, 34, 34);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Ваше устройство</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBoxUserCommand">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>370</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator:unchecked {
    image: url(:/images/CheckBoxOn.png);
}
QCheckBox::indicator:checked {
    image: url(:/images/CheckBoxOff.png);
}
QCheckBox {
    background-color: rgba(255, 255, 255, 0);
}
</string>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="SvetLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>251</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/IconSvet.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="vlazh">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>255</y>
      <width>151</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>65%</string>
    </property>
   </widget>
   <widget class="QLabel" name="temp">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>65</y>
      <width>81</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>27°</string>
    </property>
   </widget>
   <widget class="QLabel" name="co2text">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>411</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Состояние датчика углекислого газа:</string>
    </property>
   </widget>
   <widget class="QLabel" name="temptext">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>65</y>
      <width>167</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Температура:</string>
    </property>
   </widget>
   <widget class="QLabel" name="vltext">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>255</y>
      <width>138</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Влажность:</string>
    </property>
   </widget>
   <widget class="QLabel" name="co2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>420</y>
      <width>250</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string> Всё хорошо</string>
    </property>
   </widget>
   <widget class="QLabel" name="tempicon">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>50</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/thermometericon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="vlicon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/humidityicon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="co2icon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/co2icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>tempicon</zorder>
   <zorder>temptext</zorder>
   <zorder>vlazh</zorder>
   <zorder>temp</zorder>
   <zorder>vlicon</zorder>
   <zorder>vltext</zorder>
   <zorder>co2text</zorder>
   <zorder>co2icon</zorder>
   <zorder>co2</zorder>
   <zorder>HomeButton</zorder>
   <zorder>CommandsButton</zorder>
   <zorder>FanLabel</zorder>
   <zorder>CheckBoxFan</zorder>
   <zorder>UserCommandButton</zorder>
   <zorder>CheckBoxUserCommand</zorder>
   <zorder>SvetLabel</zorder>
   <zorder>CheckBoxSvet</zorder>
   <zorder>ChainikLabel</zorder>
   <zorder>CheckBoxChainik</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
